backend/database/seeders/DatabaseSeeder.php

```diff
@@ -12,6 +13,14 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
+        DB::statement('SET FOREIGN_KEY_CHECKS=0;');
```

Comment:
@ndvkhangsgt 
Please correct.
```suggestion
       Schema::disableForeignKeyConstraints();
```